setup.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+import io
+import os
+import sys
+
+from setuptools import find_packages, setup
+
+
+if sys.argv[-1] == 'publish':
+    os.system('python setup.py sdist upload')
+    os.system('python setup.py bdist_wheel upload')
+    sys.exit()
+
+
+readme = io.open('README.md', 'r', encoding='utf-8').read()
+
+setup(
+    name='storyscript-http-gateway',
```

Comment:
```suggestion
    name='story-http-gateway',
```

To keep in naming with e.g. `story-hub` or `story-runtime`.